core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportClinicalData.java

```diff
@@ -68,13 +69,23 @@
 	@Rule
 	public ExpectedException exception = ExpectedException.none();
     
-	
+	private CancerStudy cancerStudy = null;
+
+	/**
+	 * This is executed n times, for each of the n test methods below:
+	 * @throws DaoException
+	 */
 	@Before 
 	public void setUp() throws DaoException
 	{
-		DaoGeneticProfile.reCache();
-		DaoPatient.reCache();
-		DaoSample.reCache();
+        DaoCancerStudy.reCacheAll();
```

Comment:
yeah...I thought my Eclipse was using the `.editorconfig` file that is part of this project (see https://github.com/cBioPortal/cbioportal/pull/593)...but apparently something is still wrong. Anyway, I made a small fix just for this part.